src/python/dxpy/scripts/dx.py

```diff
@@ -5079,6 +5095,15 @@ def positive_integer(value):
                                                         " at https://www.nextflow.io/blog/2021/configure-git-repositories-with-nextflow.html.",
                                                    width_adjustment=-24), dest="git_credentials").completer = DXPathCompleter(classes=['file'])
 
+
+nextflow_options.add_argument('--cache-docker', help=fill("Stores a container image tarball in the currently selected project"
```

Comment:
minor: maybe add this argument with `default=False`?

Reply:
other boolean flags don't have a default. Following the convention